Sources_v3/Database/DatabaseSession.swift

```diff
@@ -99,6 +99,18 @@ protocol DatabaseSession: UserDatabaseSession, CurrentUserDatabaseSession, Messa
     ) throws -> ChannelDTO
     
     func loadChannel<ExtraData: ExtraDataTypes>(cid: ChannelId) -> ChannelModel<ExtraData>?
+    
+    // MARK: - Channel read
+    
+    @discardableResult
+    func saveChannelRead<ExtraData: ExtraDataTypes>(
+        payload: ChannelReadPayload<ExtraData>,
+        for cid: ChannelId
+    ) throws -> ChannelReadDTO
+    
+    func loadChannelRead(cid: ChannelId, userId: String) -> ChannelReadDTO?
+    
+    func loadChannelReads(for userId: UserId) -> [ChannelReadDTO]
```

Comment:
Can you move this to `ChannelReadDatabaseSession` please? The same way we have it for `UserDatabaseSession` or `MessageDatabaseSession`

Reply:
Fixed in 890b87135595133e4bedfecf0ca1d6d0d0622ce9
I wonder why don't we have `MemberDatabaseSession` and `ChannelDatabaseSession` though? I thought we add custom DatabaseSession protocols if we see the need, and I didn't see the need for `ChannelReadDatabaseSession`

Reply:
We talked about it already. Please see https://github.com/GetStream/stream-chat-swift/pull/475#discussion_r484862958 and CIS-332